editor/src/core/model/project-file-utils.ts

```diff
@@ -216,6 +218,37 @@ export const getUtopiaJSXComponentsFromSuccess = Utils.memoize(
   getUtopiaJSXComponentsFromSuccessInner,
 )
 
+export function applyUtopiaJSXComponentsChanges(
```

Comment:
I moved this verbatim from editor-state.ts
the reason was transitive imports killing the parser-printer-worker